### PR TITLE
Add History API

### PR DESCRIPTION
**Pull request by autonome**

### CODE REVIEW

File: features/history.yml.dist [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "5"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "4"
+  #   firefox_android: "4"
+  #   safari: "5"
+  #   safari_ios: "4"
+  - api.History.pushState
```
Comment (captainbrosset) [inline features/history.yml.dist:45]: This is the support data that matches caniuse.com. So, it seems like everything up to this point is really necessary to use the API, but the 2 following keys are nice-to-haves (`state` and `scrollRestoration`). Reading their documentation, it does seem like you can use the API without them.
So I would be tempted to use `compute_from` on `api.History.pushState` to match caniuse.com.

File: features/history.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: History
+description: "The History API allows manipulation of the browser session history, from navigations to state management, in the tab or frame that the current page is loaded in."
```
Comment (captainbrosset) [inline features/history.yml:2]:
```suggestion
description: "The History API manipulates the browser session history, from navigations to state management, in the tab or frame that the current page is loaded in."
```

**Review by captainbrosset — CHANGES_REQUESTED** [review]

**Review by jamesnw — APPROVED** [review]
Approved pending changes from @captainbrosset

File: features/history.yml (hunk 2 above)
Comment (autonome) [reply features/history.yml:2]: the curse of the passive voice!

**Comment by autonome** [comment]
Implemented @captainbrosset's changes, and dismissing review for merge. Github review flow isn't great for the "r+ w/ changes addressed" approach.

**Pull request merged**